catalogue/webapp/components/ImageEndpointSearchResults/ImageEndpointSearchResults.tsx

```diff
@@ -1,57 +1,120 @@
-import { FunctionComponent, useState } from 'react';
+import { FunctionComponent, useEffect, useState, useCallback } from 'react';
+import Gallery from 'react-photo-gallery';
+import styled from 'styled-components';
+
+import { Image, CatalogueResultsList } from '@weco/common/model/catalogue';
+
 import ExpandedImage from '../ExpandedImage/ExpandedImage';
 import ImageCard from '../ImageCard/ImageCard';
-import { Image, CatalogueResultsList } from '@weco/common/model/catalogue';
 import Modal from '@weco/common/views/components/Modal/Modal';
+import { convertImageUri } from '@weco/common/utils/convert-image-uri';
 
 type Props = {
   images: CatalogueResultsList<Image>;
 };
 
+const imageMargin = 16;
+
+const GalleryContainer = styled.div`
+  margin: 0 -${imageMargin / 2}px;
+
+  // This div is rendered by Gallery so we're targetting it on mobile only to ensure images are aligned on both sides
+  .react-photo-gallery--gallery > div {
+    justify-content: space-around;
+  }
+
+  @media (min-width: ${props => props.theme.sizes.medium}px) {
+    margin: 0 -${imageMargin}px;
```

Comment:
Possibly daft question: is this about making images flush with the edge of a mobile screen, or something else?

Reply:
Every image has a horizontal margin of 16px, but because I don't know which image starts and ends a row at any given time, I wanted to offset these margins on the container so it feel more flush with the sides of the rest of the site.

Reply:
Ah, gotcha. Thanks. 👍